grumphp.yml

```diff
@@ -1,17 +1,16 @@
 parameters:
-  git_dir: .
-  bin_dir: vendor/bin
-  stop_on_failure: false
-  hide_circumvention_tip: true
-  ignore_unstaged_changes: false #brocken
-  process_timeout: "%convention.process_timeout%"
   convention.process_timeout: 60
   convention.security_checker_blocking: true
   convention.jsonlint_ignore_pattern: []
   convention.xmllint_ignore_pattern: []
   convention.yamllint_ignore_pattern: []
   convention.phpcslint_ignore_pattern: []
   convention.xlifflint_ignore_pattern: ["#typo3conf/l10n/(.*)#"]
+grumphp:
+  stop_on_failure: false
```

Comment:
does this mean that the process does not break if a job fails?

Reply:
https://github.com/phpro/grumphp/blob/master/doc/parameters.md

**stop_on_failure**

Default: false

This parameter will tell GrumPHP to stop running tasks when one of the tasks results in an error. By default GrumPHP will continue running the configured tasks.